py3status/modules/external_script.py

```diff
@@ -97,9 +100,15 @@ def external_script(self):
             output = ''
 
         response['full_text'] = self.py3.safe_format(
-            self.format, {'output': output})
+            self.format, {'output': output, 'line': len(output_lines)})
         return response
 
+    def on_click(self, event):
+        button = event["button"]
+        if button != self.button_refresh:
+            self.py3.notify_user(self.output)
+            self.py3.prevent_refresh()
```

Comment:
One problem I see with this simple approach is that if a user added a custom `on_click 1` handler, this will now execute both their handler and show notification. Maybe worth adding a flag `show_click_notifications`, or instead of `button_refresh` implement `button_show_notification` (so condition changes to `if button == self.button_show_notification`). Thoughts?

Reply:
I already changed to `button_show_notification`, it's disabled by default so won't suddenly conflict with custom `on_click 1` handlers. I think we are good here.